src/v2/Apps/Collect/Routes/Collect/index.tsx

```diff
@@ -30,22 +30,19 @@ export interface CollectAppProps {
   filterArtworks: collectRoutes_ArtworkFilterQueryResponse["filterArtworks"]
 }
 
-export const CollectApp = track({
-  context_page: Schema.PageName.CollectPage,
-})((props: CollectAppProps) => {
```

Comment:
Has `context_page` migrated to `contextPageOwnerType: OwnerType.collect`?

Are we sure that this component no longer needs to be wrapped with `track` (to inject react tracking context)? (This could have been legecy code from before the new app shell.)

Reply:
Yes, the `context_page` is replaced by `contextOwnerType`. The `track` is not necessary with the `useTracking` implementation, so events are still firing as expected without the decorator.

Reply:
The purpose of that top-level `track` is to inject the context (and optionally pass along params). So at one point that was needed for this piece of the react tree, otherwise `useTracking` wouldn't have a context to pull from. (It looks like [Boot](https://github.com/artsy/force/blob/master/src/v2/Artsy/Router/Boot.tsx#L34) likely takes care of this now.)